lib/grid/index.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import css from './grid.css!'
+
+var grid = angular.module('grid', []);
+
+// COLUMN
+// ------------------------
+grid.directive('column', function column(){
```

Comment:
Named functions should be more descriptive to aid debugging of lovely js stack traces. So something like `columnDirective` or `columnDirectiveFactory`